.travis.yml

```diff
@@ -36,26 +37,25 @@ notifications:
 
 matrix:
   include:
+    # 2.7 build (until 0.14 is release)
     - os: linux
       python: 2.7
-      env: PIP_FLAGS="--pre"
+      env: WITH_PYAMG=1 QT=PyQt4
     - os: linux
       python: 2.7
-      env: WITH_PYAMG=1 MINIMUM_REQUIREMENTS=1 WITH_QT=1
+      env: MINIMUM_REQUIREMENTS=1
     - os: linux
       python: 3.4
-      env: OPTIONAL_DEPS=1 WITH_PYSIDE=1 BUILD_DOCS=1 DEPLOY_DOCS=1
-    - os: linux
-      python: 3.5
     - os: linux
       python: 3.5
-      env: PIP_FLAGS="--pre"
```

Comment:
@hmaarrfk, @stefanv makes a good point: at least one of our builds should use `--pre`... I suggest 3.6.

Reply:
Yeah, I didn't understand why were were testing for `--pre`.

I was mostly trimming builds to test quickly.

For now, I also added it to 2.7